index.js

```diff
@@ -0,0 +1,90 @@
+$(document).ready(function() {
+  $('#allTrips').on('click', function() {
```

Comment:
If you're not going to use Arrow functions, it's good to name the functions like:
`$('#allTrips').on('click', function loadTrips() {`